### PR TITLE
Updated examples to use the "expect" RSpec syntax.

### PR DESCRIPTION
**Pull request by jgebal**
The examples were using deprecated `.should` syntax
This pull request is to address that.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,4 @@
 sqlnet.log
 Gemfile.lock
 /spec/sandbox
+.idea
```
Comment (javornikolov) [inline .gitignore]: May we move that to a separate PR (maybe we can also think of other commonly ignored files).

**Comment by jgebal** [comment]
Change to `.gitignore` removed

**Comment by javornikolov** [comment]
Thanks! Merging...
